apps/web/src/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationCard.tsx

```diff
@@ -41,92 +45,113 @@ const ApplicationCard = ({
     application.status,
     application.archivedAt,
   );
+  const isApplicantPlaced = isPlaced(application.status);
   const applicationCanBeDeleted = canBeDeleted(application.status);
-
-  let borderKey: BorderMapKey = "dt-gray";
-  if (!application.archivedAt && application.status) {
-    borderKey = borderKeyMap[application.status];
-  }
-
-  const border = borderMap[borderKey];
+  const recruitmentIsExpired = isExpired(
+    application.poolAdvertisement?.closingDate,
+  );
+  const submittedAt = formatSubmittedAt(application.submittedAt, intl);
```

Comment:
submittedAt is not passed into this, it is missing from the graphql operation
https://github.com/GCTC-NTGC/gc-digital-talent/blob/4e0cfeb633af5624ac6deb28621d28b21679aca5/apps/web/src/pages/Applications/MyApplicationsPage/applicationOperations.graphql#L1-L39

resulting in 
![image](https://user-images.githubusercontent.com/40485260/219790566-83fd8a29-f62c-48a7-9167-2ca75c26f455.png)

Reply:
Good catch! Added in [add submittedAt to application query](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5744/commits/251c4c02883c09a759aeeb39a2bff11c3d8796c2)